src/mcom/manage-stock-aggregates.md

```diff
@@ -0,0 +1,30 @@
+---
+ee_only: true
+title: Manage Stock Aggregates
+---
+
+Manage Stock Aggregates provides information about existing stock aggregates, and allows you to add new stock aggregates or edit or delete existing ones.
+
+![]({% link images/images-ee/mcom-manage-stock-aggregates.png %}){: .zoom}
+_Manage Stock Aggregates_
+
+You [create sources](https://omsdocs.magento.com/en/features-processes/stock-sourcing/inventory/#configure-sources) and [aggregates](https://omsdocs.magento.com/en/features-processes/stock-sourcing/inventory/#configure-stock-aggregates) in the Order Management System (OMS) Admin. Then you assign which website(s) are associated with each aggregate via Manage Stock Aggregates in the Magento Commerce (MC) Admin.
+
+## Add a new stock aggregate
+
+1. On the _Admin_ sidebar, go to **MCOM** > **Manage Stock Aggregates**.
+1. Click **Add New Stock Aggregate**.
+1. Enter a **Stock code** (name) for this stock aggregate.
+
+   {:.bs-callout-info}
+   The stock code you enter should match the [stock aggregate code](https://omsdocs.magento.com/en/features-processes/stock-sourcing/inventory/#configure-stock-aggregates) in the OMS.
+
+1. Choose a **Website** from the populated list.
+1. When complete, click **Save Stock Aggregate**.
+
+## Edit or delete a stock aggregate
+
+1. On the _Admin_ sidebar, go to **MCOM** > **Manage Stock Aggregates**.
+1. In the Action column, click **Edit** for an existing stock aggregate.
```

Comment:
This is the convention we have been using for locating a column or section on the page.

```suggestion
1. In the _Action_ column, click **Edit** for an existing stock aggregate.
```